Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx

```diff
@@ -246,6 +248,34 @@ export function CreateOptions(props: CreateOptionsProps) {
     }
   }, [templates]);
 
+  useEffect(() => {
+    if (props.location?.search) {
+      const decoded = decodeURIComponent(props.location.search);
+      const { source, payload } = querystring.parse(decoded);
+      if (payload && typeof payload === 'string' && source === 'abs') {
+        const profile = JSON.parse(payload);
+        const alias = `abs-${profile.botName}-${profile.appId}`;
+        // check to see if Composer currently has a bot project corresponding to the alias
+        axios
+          .get<any>(`/api/projects/alias/${alias}`)
+          .then((aliasRes) => {
+            console.log(aliasRes);
+            if (aliasRes.status === 200) {
+              navigate(`/bot/${aliasRes.data.id}`);
+              return;
+            }
+          })
+          .catch((e) => {
+            setIsOpen(true);
+          });
+      } else {
+        setIsOpen(true);
+      }
+    } else {
+      setIsOpen(true);
+    }
```

Comment:
can this be simplified to:

```
if (props.location.search) {

  ...

  if (payload && typeof payload === 'string' && source === 'abs') {
  
    ...
  
    axios.get
      .then(
        ...
        navigate
        return
      )
      .catch(setIsOpen(true))
   }
}
setIsOpen(true)

```

It seems like the two `else` blocks are redundant since we only want to navigate if we find a matching alias, otherwise we want to `setIsOpen(true)`